.github/workflows/test.yml

```diff
@@ -62,19 +62,56 @@ jobs:
   test_local_cluster_with_prometheus_grafana:
```

Comment:
nit: `test_local_cluster_with_prometheus_grafana` should be `test_local_cluster_with_prometheus_grafana_tekton`?